packages/components/src/Chip/Chip.css

```diff
@@ -50,11 +55,6 @@
   justify-content: center;
 }
 
-.chip .suffix:hover,
```

Comment:
Chip prefix and suffix do not currently have the ability to receive interactions and should not have a hover state.